typescript/keys.d.ts

```diff
@@ -13,3 +13,20 @@ export interface Key {
   }
   isSolutions: boolean
 }
+
+export type DeveloperKeyAccess = {
+  submissions?: {
+    create: {
+      formIds: number[]
```

Comment:
Isn't it this, or did it evolve and I wasn't paying attention?
```suggestion
      formIds?: number[] | boolean
```

Reply:
We want the data structure to cable of supporting  a boolean in the future. But until we have to, I dont want to implement the boolean yet. It would be a feature that potentially never gets used if we implemented it now.